consensus/ibft/ibft.go

```diff
@@ -1233,6 +1244,19 @@ func (i *Ibft) verifyHeaderImpl(snap *Snapshot, parent, header *types.Header) er
 		return err
 	}
 
+	// verify last committed seals
+	if parent.Number >= 1 {
+		// find validators who validated last block
+		parentSnap, err := i.getSnapshot(parent.Number - 1)
```

Comment:
Doesn't this mean we're fetching the snapshot of the grandparent (`parent.Number - 1`) ?

Reply:
Yes, you're right. Snapshot for n-th block means a snapshot that has been updated by block at n. We want to verify parent block, which means we need to get the snapshot for grandparent block.